### PR TITLE
ERMS-3336

### PR DESCRIPTION
**Pull request by Inselhopper**

### CODE REVIEW

File: grails-app/views/survey/_costItemModal.gsp [hunk 1]
```diff
@@ -99,7 +99,7 @@
                                        number="${costItem?.costInBillingCurrency}"
                                        minFractionDigits="2" maxFractionDigits="2"/>"/>
 
-                        <g:select class="ui dropdown dk-width-auto" name="newCostCurrency" title="${message(code: 'financials.addNew.currencyType')}"
+                        <g:select class="ui dropdown la-small-dropdown la-not-clearabl" name="newCostCurrency" title="${message(code: 'financials.addNew.currencyType')}"
```
Comment (croqueGrec09) [inline grails-app/views/survey/_costItemModal.gsp:102]: class typo la-not-clearable

**Review by croqueGrec09 — COMMENTED** [review]
one remark

File: grails-app/views/survey/_costItemModal.gsp (hunk 1 above)
Comment (Inselhopper) [reply grails-app/views/survey/_costItemModal.gsp:102]: oh, copy paste fehler

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**